blogs/index.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+---
+  <div>
+    <h1 style="text-align: center;"> Latest Blogs</h1>
+      <div style="margin:20px 150px 40px 150px; font-size: 22px; border-style: solid; padding: 15px; ">
```

Comment:
as blogs will have many other CSS. Try adding it in a class and also add it in assets/css/blog.css. Make a new folder